api/types/network/summary.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package network
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// Summary Network list response item
+//
+// swagger:model Summary
+type Summary struct {
+	Network
+}
+
+// UnmarshalJSON unmarshals this object from a JSON structure
+func (m *Summary) UnmarshalJSON(raw []byte) error {
+	// AO0
+	var aO0 Network
+	if err := swag.ReadJSON(raw, &aO0); err != nil {
+		return err
+	}
+	m.Network = aO0
+
+	return nil
+}
+
+// MarshalJSON marshals this object to a JSON structure
+func (m Summary) MarshalJSON() ([]byte, error) {
+	_parts := make([][]byte, 0, 1)
+
+	aO0, err := swag.WriteJSON(m.Network)
+	if err != nil {
+		return nil, err
+	}
+	_parts = append(_parts, aO0)
+	return swag.ConcatJSON(_parts...), nil
```

Comment:
Ah, I see now what you meant!

But curious; why were these custom MarshalJSON functions needed? Wouldn't our existing code handle the embedded structs just find? Would it still be possible to only produce the bare structs?

Reply:
Success! I was able to suppress all the unwanted functions.